### DISCUSSION TITLE
？？？？

### DISCUSSION THREAD

**Discussion by Luna-Sec** [lencx/ChatGPT#1217] [General]
我不懂为什么gpt突然问不了，就能发没回复？然后重装重新登录一直提示密码错误，我错尼玛啊？ri他马的我谷歌账号能登录，这gpt不就是谷歌账号吗？什么玩意。我还以为密码记错了一直试，现在gpt账号都冻结了，ri尼玛的艹

**Comment by wang-CC0711** [comment]
我也是,搞得我又回到网页版本了..

**Comment by lencx** [comment]
请文明发言，在这里骂娘屁用没有：
<img width="622" alt="Screenshot 2024-03-09 at 14 35 22" src="https://github.com/lencx/ChatGPT/assets/16164244/53acafce-13ba-4835-98b4-5c795dbc17bf">

**Reply by tianzhi0hao** [reply]
感谢！找到问题原因了，还以为被墙了。还是喜欢比较简单的Chatgpt客户端
